checks.d/veneur.py

```diff
@@ -25,11 +25,13 @@ def check(self, instance):
             builddate = datetime.datetime.fromtimestamp(int(r.text))
 
             tdelta = datetime.datetime.now() - builddate
-            self.histogram(self.BUILDAGE_METRIC_NAME, tdelta.seconds)
```

Comment:
Does this also need the `instance`'s tags?

Reply:
I think those get set by default? Looking at `/metric/summary?filter=build_age`.

Reply:
You applied them to the gauge below though?

Reply:
Because otherwise it seems we overwrite them when passing a new value for `tags`. The default is safe, though.